cmd/main.go

```diff
@@ -753,7 +753,18 @@ func createControllerManager() (manager.Manager, error) {
 			NewCache: cache.BuilderWithOptions(cache.Options{
 				SelectorsByObject: cache.SelectorsByObject{
 					&corev1.Secret{}: {
-						Label: labels.SelectorFromSet(labels.Set{controllers.SecretLabelName: controllers.SecretLabelValue}),
+						Label: labels.SelectorFromSet(
+							labels.Set{
+								controllers.SecretLabelName: controllers.SecretLabelValue,
+							},
+						),
+					},
+					&corev1.Secret{}: {
+						Label: labels.SelectorFromSet(
+							labels.Set{
+								"hive.openshift.io/secret-type": "kubeconfig",
```

Comment:
What do you think we construct the label/value using [Hive's constants](https://github.com/openshift/hive/blob/5990617a5f027cc834f1b14d6a0ece0c7dc35696/pkg/installmanager/installmanager.go#L1146)? 

This should help us stay closer to what they are doing if any of these values are changed in the future.

Reply:
Sure, will update.